rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -341,12 +341,13 @@ void game_list_frame::Refresh(const bool from_drive, const bool scroll_after)
 		m_notes.clear();
 		m_games.pop_all();
 
-		m_progress_dialog = new progress_dialog(tr("Loading games"), tr("Loading games, please wait..."), tr("Cancel"), 0, 0, true, this, Qt::Dialog | Qt::WindowTitleHint | Qt::CustomizeWindowHint);
+		m_progress_dialog = new progress_dialog(tr("Loading games"), tr("Loading games, please wait..."), tr("Cancel"), 0, 0, false, this, Qt::Dialog | Qt::WindowTitleHint | Qt::CustomizeWindowHint);
 
 		connect(m_progress_dialog, &QProgressDialog::finished, this, [this]()
 		{
 			if (m_progress_dialog == QObject::sender())
 			{
+				m_progress_dialog->deleteLater();
```

Comment:
wth is this ?

Reply:
idk but it does seem to fix the random crashes on startup, maybe I should move it to progress_dialog.cpp

Reply:
It's wrong though

Reply:
QProgressDialog::SetValue is known to process events internally if the the dialog is is in modal, I think the issue is that it deletes itself before SetValue returns.
We should probably postpone deletion manually instead of relying on auto-delete.

Reply:
You don't understand. Finished is not necessarily called when the dialog is destroyed.

Reply:
Anyway, this doesn't belong in some spu PR.